frontend/src/lib/api.ts

```diff
@@ -765,6 +766,21 @@ class ApiRequest {
         return this.integrations(teamId).addPathComponent(id).addPathComponent('channels')
     }
 
+    public integrationGoogleAdsAccounts(id: IntegrationType['id'], teamId?: TeamType['id']): ApiRequest {
+        return this.integrations(teamId).addPathComponent(id).addPathComponent('accessibleAccounts')
```

Comment:
All api endpoints should be snake case so `accessible_accounts` (also see my point about prefixing with googleads)

Reply:
done - [`efcda05` (#26088)](https://github.com/PostHog/posthog/pull/26088/commits/efcda05b6659b0be4c8636ab7f2164dd12356b31)